src/components/home/slider/sliderChildren.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react'
+import {FC} from 'react'
+import SliderModal from './SliderModal'
+import {Icon} from 'react-icons-kit'
+import {chevronLeft, chevronRight} from 'react-icons-kit/feather'
+import {Movie} from './SliderModal'
+
+import * as SliderParent from './style'
+
+interface Props {
+  translateValue: number
+  backImg(): void
+  movies: Movie[]
+  activeIndex: number
+  nextImg(): void
+}
+
+const SliderChildren: FC<Props> = ({
+  translateValue,
+  backImg,
+  movies,
+  activeIndex,
+  nextImg,
+}) => {
+  return (
+    <SliderParent.SliderParent>
+      <SliderParent.Welcome>
+        Your weekend buddy for this week
+      </SliderParent.Welcome>
+      <SliderParent.SliderTop>
+        <SliderParent.SliderWrapper transform={translateValue}>
+          {movies.map((item, key) => (
```

Comment:
It's better to use the `id` of `item`. I don't think you have it in the type definition but I think you should include it when getting the data from the web.

Reasons can be read here https://www.robinwieruch.de/react-list-key

Reply:
It wasn't supported by the old API, Now I have `id`

will use it in the next fix